### ISSUE TITLE
second AnimatedSprite won't show correct sprites from TextureAtlas

### ISSUE THREAD

**Issue by crumbgit** [craftworkgames/MonoGame.Extended#252]
Hi, 

I've been trying out monogame.extended and finding it very good on the whole. Though I have come across something strange.

I've got a TextureAtlas feeding into the main character AnimatedSprite and it works perfectly. But if I use a second or more AnimatedSprite it will only show the sprite from index 0 of the atlas.
For example

```
private AnimatedSprite _bonoSprite;
private AnimatedSprite _testSprite;

var bonoTexture = Content.Load<Texture2D>("Tilesets/SpritesFull");
var bonoAtlas = TextureAtlas.Create(bonoTexture, 32, 32);
var bonoAnimationFactory = new SpriteSheetAnimationFactory(bonoAtlas);

bonoAnimationFactory.Add("walkSouth", new SpriteSheetAnimationData(new[] { 40, 41, 40, 42 }, isLooping: false));
bonoAnimationFactory.Add("walkWest", new SpriteSheetAnimationData(new[] { 52, 51, 50, 49 }, isLooping: false));

_bonoSprite = new AnimatedSprite(bonoAnimationFactory, "walkSouth");
_bonoSprite.Position = new Vector2(144, 272);

_testSprite= new AnimatedSprite(bonoAnimationFactory, "walkWest");
_testSprite.Position = new Vector2(176, 304);

<draw code>
```
_bonoSprite displays as expected, _testSprite regardless of the animation set only shows texture index 0.

I originally had a new TextureAtlas from a different content file, that only showed index 0, then i actually moved the sprites I wanted into the same content, to use the same textureatlas that already works. still only shows sprite 0!.

**Comment by crumbgit** [comment]
took me nearly two days but finally figured it out...

the draw code had the `.update(deltaseconds)` missing for the second sprite. Works fine now.

One extra question, is it possible to set an individual TiledTile to not be visible? should i start a new thread for questions? i've got a few :)

**Comment by Aurioch** [comment]
Judging on the quick look at the code, I don't think so.

However, judging on the quick look at the code, there is a workaround using `TiledTileSetTile.Properties`, which is actually nothing more than simple `Dictionary<string, string>`. If I figured this out correctly (don't take my word for it, I'm very possibly completely wrong), `TiledProperties` class holds property for tile specified in Tiled.

**Comment by craftworkgames** [comment]
> One extra question, is it possible to set an individual TiledTile to not be visible?

The Tiled renderer in the `develop` branch is going through a lot of changes at the moment. I'm not entirely certain what features have been completely implemented.

> should i start a new thread for questions? i've got a few :)

There's no strict rules about what to ask where, but I would prefer to keep each thread as a single topic if possible. As a bit of a guideline we've got a number of places to ask questions:

 - Use the [Gitter chat](https://gitter.im/craftworkgames/MonoGame.Extended) for quick questions.
 - We've got an [Extended category](http://community.monogame.net/c/extended) in the official MonoGame forums. That's a good place for general question asking.
 - We've even got a [tag on stackoverflow](http://gamedev.stackexchange.com/questions/tagged/monogame-extended) ;)
 - Obviously, here on github we also discuss a lot of things like new features, major changes and you know.. issues.
